apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/class-wp-rest-wpcom-block-editor-nux-status-controller.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * WP_REST_WPCOM_Block_Editor_NUX_Status_Controller file.
+ *
+ * @package A8C\FSE
+ */
+
+namespace A8C\FSE;
+
+/**
+ * Class WP_REST_WPCOM_Block_Editor_NUX_Status_Controller.
+ */
+class WP_REST_WPCOM_Block_Editor_NUX_Status_Controller extends \WP_REST_Controller {
```

Comment:
This is throwing a duplicate class name warning in PHPCS, because it shares the same name as the class defined in `wpcom-block-editor-nux` (https://github.com/automattic/wp-calypso/blob/trunk/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/class-wp-rest-wpcom-block-editor-nux-status-controller.php)

Is there a clean way we could share the one file and class between these two areas, to remove the duplication? It looks like they're very similar.

From PHPCS:

```
----------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------
 13 | WARNING | Duplicate class name
    |         | "A8C\FSE\WP_REST_WPCOM_Block_Editor_NUX_Status_Controller"
    |         | found; first defined in
    |         | /home/runner/work/wp-calypso/wp-calypso/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/class-wp-rest-wpcom-block-editor-nux-status-controller.php
    |         | on line 13
    |         | (Generic.Classes.DuplicateClassName.Found)
----------------------------------------------------------------------
```

Reply:
I removed the duplicate file, keeping apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/class-wp-rest-wpcom-block-editor-nux-status-controller.php.  ANd then updated the file path where the file is called in class-wpcom-block-editor-nux.php.

Does that seem like a sensible solution?